### ISSUE TITLE
Support for MS SQL Server

### ISSUE THREAD

**Issue by danilvoe** [elastic/apm-agent-ruby#297]
I want to see support for calls to MS SQL Server. The project has a call to the database MS SQL Server to get data. Gem  used tiny_tds

**Comment by mikker** [comment]
Hi @danilvoe. Is the app a Rails app using ActiveRecord? If it is then MS SQL Server should be supported through that.

**Comment by danilvoe** [comment]
No, the application itself does not use MS SQL; several methods in the code request information from MS SQL directly. gem used  https://github.com/rails-sqlserver/tiny_tds

**Comment by mikker** [comment]
Got it. I'll have to see if I can find the time. Otherwise you're very welcome to submit a PR with a _Spy_.

See [the others for reference](https://github.com/elastic/apm-agent-ruby/tree/master/lib/elastic_apm/spies)
